.gitignore

```diff
@@ -44,7 +170,6 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
```

Comment:
I removed this from the "master" gitignore, since it ignored the necessary lib folder from Flutter